src/style.css

```diff
@@ -0,0 +1,116 @@
+* {
+  margin: 0;
+}
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  height: 406px;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  margin: 32px auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__headline {
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  line-height: 18px;
+  margin: 0 16px 4px;
```

Comment:
after you add padding for the card adding margin-bottom for each element would be pretty enough